### DISCUSSION TITLE
0.32.0 version Kraft and Zookeeper

### DISCUSSION THREAD

**Discussion by bahaeddinoz** [strimzi/strimzi-kafka-operator#7591] [Q&A] [answered]
Hi,
Would like to ask about latest version and Zookeeper support. Does latest version 0.32.0 works with Kraft feature and no need zookeeper any more ? When upgrading from previous versions 0.31.0 what will do to the zookeepers ?

Thank in advance for your helps 🙏

**Comment by scholzj** [comment, marked as answer]
Strimzi currently has a feature gate which you have to enable to run Kafka in the KRaft mode. But it is experimental and not production ready. While Kafka called the KRaft mode _production ready for new clusters_, it is still missing many important features such as JBOD storage, upgrades from ZooKeeper or for example SCRAM-SHA authentication. As these are implemented, better support in Strimzi will be added as well. 

So when you upgrade to Strimzi 0.32 / Kafka 3.3.1, basically nothing changes compared to previous releases.

**Reply by bahaeddinoz** [reply]
thanks a lot for quick answer @scholzj 🙏
